From f6b553a06ed9d14f8007c1a17b848bb204fd71f4 Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Wed, 10 Jun 2015 12:47:30 +0200
Subject: [PATCH 08/19] Remove obsolete theme.

Activities using it will crash with an IllegalStateException stating
that an AppCompat theme needs to be used.

Change-Id: I09acbe9cc0cc9fe1653044194ae4822c923bf987
---
 AndroidManifest.xml       |  4 ++--
 res/values-v21/themes.xml | 12 ------------
 res/values/themes.xml     |  9 ---------
 3 files changed, 2 insertions(+), 23 deletions(-)
 delete mode 100644 res/values-v21/themes.xml

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index b0bb881d9..83504b47d 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -32,7 +32,7 @@
     <application
         android:icon="@mipmap/ic_launcher_mail"
         android:label="@string/app_name"
-        android:theme="@style/UnifiedEmailTheme"
+        android:theme="@style/UnifiedEmailTheme.Appcompat"
         android:supportsRtl="false" >
         <!-- Enable search in all activities -->
         <meta-data android:name="android.app.default_searchable"
@@ -112,7 +112,7 @@
         <activity
                 android:name=".browse.EmlViewerActivity"
                 android:label="@string/app_name"
-                android:theme="@style/UnifiedEmailTheme" >
+                android:theme="@style/UnifiedEmailTheme.Appcompat" >
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
diff --git a/res/values-v21/themes.xml b/res/values-v21/themes.xml
deleted file mode 100644
index a25b42836..000000000
--- a/res/values-v21/themes.xml
+++ /dev/null
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <style name="UnifiedEmailTheme" parent="@android:style/Theme.Material.Light.DarkActionBar">
-        <item name="android:actionModeBackground">@color/action_mode_background</item>
-        <item name="android:actionOverflowButtonStyle">@style/ActionBarOverflowButtonStyle</item>
-        <item name="android:listViewWhiteStyle">@android:style/Widget.Material.Light.ListView</item>
-        <item name="android:colorPrimary">@color/primary_color</item>
-        <item name="android:colorPrimaryDark">@color/primary_dark_color</item>
-        <item name="android:colorAccent">@color/accent_blue</item>
-        <item name="android:colorEdgeEffect">@color/edge_effect_color</item>
-    </style>
-</resources>
\ No newline at end of file
diff --git a/res/values/themes.xml b/res/values/themes.xml
index 2cee63872..550f1f5ec 100644
--- a/res/values/themes.xml
+++ b/res/values/themes.xml
@@ -1,14 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <style name="UnifiedEmailTheme" parent="@android:style/Theme.Holo.Light.DarkActionBar">
-        <item name="android:actionBarStyle">@style/ActionBarStyle</item>
-        <item name="android:actionBarWidgetTheme">@style/ActionBarWidgetTheme</item>
-        <item name="android:actionModeBackground">@color/action_mode_background</item>
-        <item name="android:actionOverflowButtonStyle">@style/ActionBarOverflowButtonStyle</item>
-        <item name="android:homeAsUpIndicator">@drawable/ic_arrow_back_wht_24dp_with_rtl</item>
-        <item name="android:listViewWhiteStyle">@android:style/Widget.Holo.Light.ListView</item>
-    </style>
-
     <style name="ActionBarWidgetTheme" parent="@android:style/Theme.Holo">
         <item name="android:dropDownListViewStyle">@style/ActionBarDividerlessListViewStyle</item>
         <item name="android:popupMenuStyle">@android:style/Widget.Holo.Light.ListPopupWindow</item>
-- 
2.17.1

